spec/facility_spec.rb

```diff
@@ -20,7 +33,160 @@
       @facility.add_service('New Drivers License')
       @facility.add_service('Renew Drivers License')
       @facility.add_service('Vehicle Registration')
+
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe "#register_vehicle" do 
+    it 'has a Vehicle Registration service' do 
+      expect(@cruz).to be_an_instance_of(Vehicle)
+      expect(@facility.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+      expect(@facility.registered_vehicles).to eq([])
+      expect(@facility.collected_fees).to eq(0) 
+      expect(@cruz.registration_date).to eq(nil)
+    end
+  end
+
+  describe "#register_vehicle cruz" do 
+    it 'can register a specific vehicle, regular type' do 
+      expect(@cruz).to be_an_instance_of(Vehicle)
+      expect(@facility.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+      @facility.register_vehicle(@cruz)
+
+      expect(@facility.registered_vehicles).to eq([@cruz]) 
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@facility.collected_fees).to eq(100) 
+      expect(@cruz.registration_date).to eq(Date.today)
+    end
+  end
+
+  describe "#register_vehicle camaro" do 
+    it 'can register a specific vehicle, antique type' do 
+      expect(@camaro).to be_an_instance_of(Vehicle)
+      expect(@facility.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+      @facility.register_vehicle(@camaro)
+
+      expect(@camaro.plate_type).to eq(:antique)
```

Comment:
So this line is passing because it's calling the method of `plate_type` in the Vehicle class, however this is not happening in the register_vehicle method.